diff --git a/tests/Controller/ProductControllerTest.php b/tests/Controller/ProductControllerTest.php
index 6e83b4e..33f6f28 100644
--- a/tests/Controller/ProductControllerTest.php
+++ b/tests/Controller/ProductControllerTest.php
@@ -23,7 +23,6 @@ class ProductControllerTest extends PantherTestCase
             ->fillField('Name', 'Slightly old popcorn')
             ->click('Update')
             ->followRedirect()
-            ->dd()
             ->assertSuccessful();
 
         $this->assertSame('Slightly old popcorn', $product->getName());
