diff --git a/phpunit.xml.dist b/phpunit.xml.dist
index 8d9d7f6..27c68d0 100644
--- a/phpunit.xml.dist
+++ b/phpunit.xml.dist
@@ -33,6 +33,6 @@
 
     <!-- Run `composer require symfony/panther` before enabling this extension -->
     <extensions>
-        <extension class="Symfony\Component\Panther\ServerExtension" />
+        <extension class="Zenstruck\Browser\Test\BrowserExtension" />
     </extensions>
 </phpunit>
