diff --git a/tests/Controller/ProductControllerTest.php b/tests/Controller/ProductControllerTest.php
index 18b1139..33f6f28 100644
--- a/tests/Controller/ProductControllerTest.php
+++ b/tests/Controller/ProductControllerTest.php
@@ -4,26 +4,27 @@ namespace App\Tests\Controller;
 
 use App\Factory\ProductFactory;
 use Symfony\Component\Panther\PantherTestCase;
+use Zenstruck\Browser\Test\HasBrowser;
 use Zenstruck\Foundry\Test\Factories;
 use Zenstruck\Foundry\Test\ResetDatabase;
 
 class ProductControllerTest extends PantherTestCase
 {
-    use Factories, ResetDatabase;
+    use Factories, ResetDatabase, HasBrowser;
 
     public function testEditProduct(): void
     {
-        $client = static::createClient();
         $product = ProductFactory::createOne(['name' => 'Brand new popcorn'])
             ->enableAutoRefresh();
-        $crawler = $client->request('GET', sprintf('/product/%s/edit', $product->getId()));
 
-        $client->submitForm('Update', [
-            'product[name]' => 'Slightly old popcorn'
-        ]);
-        $client->followRedirect();
+        $this->browser()
+            ->visit(sprintf('/product/%s/edit', $product->getId()))
+            ->assertSuccessful()
+            ->fillField('Name', 'Slightly old popcorn')
+            ->click('Update')
+            ->followRedirect()
+            ->assertSuccessful();
 
-        $this->assertResponseIsSuccessful();
         $this->assertSame('Slightly old popcorn', $product->getName());
     }
 
