diff --git a/src/Factory/CategoryFactory.php b/src/Factory/CategoryFactory.php
index 326fac6..211f94a 100644
--- a/src/Factory/CategoryFactory.php
+++ b/src/Factory/CategoryFactory.php
@@ -31,7 +31,7 @@ final class CategoryFactory extends ModelFactory
     protected function getDefaults(): array
     {
         return [
-            // TODO add your default values here (https://github.com/zenstruck/foundry#model-factories)
+            'name' => self::faker()->word,
         ];
     }
 
diff --git a/src/Factory/ProductFactory.php b/src/Factory/ProductFactory.php
index d575c97..33ef73d 100644
--- a/src/Factory/ProductFactory.php
+++ b/src/Factory/ProductFactory.php
@@ -31,7 +31,13 @@ final class ProductFactory extends ModelFactory
     protected function getDefaults(): array
     {
         return [
-            // TODO add your default values here (https://github.com/zenstruck/foundry#model-factories)
+            'name' => self::faker()->words(3, true),
+            'description' => self::faker()->paragraph,
+            'brand' => self::faker()->company,
+            'price' => self::faker()->numberBetween(1000, 10000),
+            'imageFilename' => 'floppy-disc.png',
+            // will create a random category
+            'category' => CategoryFactory::new(),
         ];
     }
 
