diff --git a/tests/Controller/ProductControllerTest.php b/tests/Controller/ProductControllerTest.php
index c69474f..93ac4c4 100644
--- a/tests/Controller/ProductControllerTest.php
+++ b/tests/Controller/ProductControllerTest.php
@@ -16,7 +16,7 @@ class ProductControllerTest extends WebTestCase
         $client = static::createClient();
         $product = ProductFactory::new()->create([
             'name' => 'Brand new popcorn'
-        ]);
+        ])->enableAutoRefresh();
         $crawler = $client->request('GET', sprintf('/product/%s/edit', $product->getId()));
 
         $client->submitForm('Update', [
