diff --git a/tests/Controller/ProductControllerTest.php b/tests/Controller/ProductControllerTest.php
index 235012a..a30c7c2 100644
--- a/tests/Controller/ProductControllerTest.php
+++ b/tests/Controller/ProductControllerTest.php
@@ -2,6 +2,7 @@
 
 namespace App\Tests\Controller;
 
+use App\Factory\ProductFactory;
 use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
 use Zenstruck\Foundry\Test\Factories;
 use Zenstruck\Foundry\Test\ResetDatabase;
@@ -10,12 +11,18 @@ class ProductControllerTest extends WebTestCase
 {
     use Factories, ResetDatabase;
 
-    public function testSomething(): void
+    public function testEditProduct(): void
     {
         $client = static::createClient();
-        $crawler = $client->request('GET', '/');
+        $product = ProductFactory::createOne(['name' => 'Brand new popcorn']);
+        $crawler = $client->request('GET', sprintf('/product/%s/edit', $product->getId()));
+
+        $client->submitForm('Update', [
+            'product[name]' => 'Slightly old popcorn'
+        ]);
+        $client->followRedirect();
 
         $this->assertResponseIsSuccessful();
-        $this->assertSelectorTextContains('h1', 'Hello World');
+        $this->assertSame('Slightly old popcorn', $product->getName());
     }
 }
