diff --git a/tests/Repository/ProductRepositoryTest.php b/tests/Repository/ProductRepositoryTest.php
new file mode 100644
index 0000000..0e06956
--- /dev/null
+++ b/tests/Repository/ProductRepositoryTest.php
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Tests\Repository;
+
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+
+class ProductRepositoryTest extends KernelTestCase
+{
+    public function testSomething(): void
+    {
+        $kernel = self::bootKernel();
+
+        $this->assertSame('test', $kernel->getEnvironment());
+        //$routerService = self::$container->get('router');
+        //$myCustomService = self::$container->get(CustomService::class);
+    }
+}
