diff --git a/tests/Repository/ProductRepositoryTest.php b/tests/Repository/ProductRepositoryTest.php
index db1edb7..76f8be1 100644
--- a/tests/Repository/ProductRepositoryTest.php
+++ b/tests/Repository/ProductRepositoryTest.php
@@ -2,16 +2,27 @@
 
 namespace App\Tests\Repository;
 
+use App\Factory\ProductFactory;
 use App\Repository\ProductRepository;
 use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Zenstruck\Foundry\Test\Factories;
 
 class ProductRepositoryTest extends KernelTestCase
 {
+    use Factories;
+
     public function testSearch(): void
     {
         $kernel = self::bootKernel();
 
         $productRepository = self::$container->get(ProductRepository::class);
         $this->assertInstanceOf(ProductRepository::class, $productRepository);
+
+        ProductFactory::createOne(['name' => 'floppy disk']);
+        ProductFactory::createOne(['name' => 'popcorn']);
+        ProductFactory::createOne(['description' => 'A CD (compact disk)']);
+
+        $results = $productRepository->search('disk');
+        $this->assertCount(2, $results);
     }
 }
