diff --git a/tests/Controller/ProductControllerTest.php b/tests/Controller/ProductControllerTest.php
index 5df3c71..5a02af2 100644
--- a/tests/Controller/ProductControllerTest.php
+++ b/tests/Controller/ProductControllerTest.php
@@ -3,11 +3,11 @@
 namespace App\Tests\Controller;
 
 use App\Factory\ProductFactory;
-use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+use Symfony\Component\Panther\PantherTestCase;
 use Zenstruck\Foundry\Test\Factories;
 use Zenstruck\Foundry\Test\ResetDatabase;
 
-class ProductControllerTest extends WebTestCase
+class ProductControllerTest extends PantherTestCase
 {
     use Factories, ResetDatabase;
 
@@ -29,7 +29,7 @@ class ProductControllerTest extends WebTestCase
 
     public function testSearchAutoSuggestion(): void
     {
-        $client = static::createClient();
+        $client = static::createPantherClient();
         ProductFactory::createOne(['name' => 'Floppy Disk']);
         ProductFactory::createOne(['name' => 'Compact Disk']);
         $crawler = $client->request('GET', '/');
@@ -37,9 +37,12 @@ class ProductControllerTest extends WebTestCase
         $searchForm = $crawler->selectButton('Search')->form();
         $searchForm->setValues(['q' => 'dis']);
 
+        $client->takeScreenshot(__DIR__.'/../../screenshot.png');
+
         $this->assertCount(
             2,
             $crawler->filter('.search-preview .list-group-item')
         );
+
     }
 }
