diff --git a/tests/Controller/ProductControllerTest.php b/tests/Controller/ProductControllerTest.php
index 9767e3a..663f89d 100644
--- a/tests/Controller/ProductControllerTest.php
+++ b/tests/Controller/ProductControllerTest.php
@@ -38,6 +38,9 @@ class ProductControllerTest extends PantherTestCase
         $searchForm = $crawler->selectButton('Search')->form();
         $searchForm->setValues(['q' => 'dis']);
 
+        // wait for the AJAX to load
+        $client->waitForElementToContain('.search-preview', 'Disk');
+
         $this->assertCount(
             2,
             $crawler->filter('.search-preview .list-group-item')
